task/sast-snyk-check/0.4/sast-snyk-check.yaml

```diff
@@ -313,7 +313,10 @@ spec:
             echo "Selecting auth"
             select-oci-auth "${IMAGE_URL}" > "${HOME}/auth.json"
             echo "Attaching to ${IMAGE_URL}"
-            retry oras attach --no-tty --registry-config "$HOME/auth.json" --artifact-type "${MEDIA_TYPE}" "${IMAGE_URL}@${IMAGE_DIGEST}" "${UPLOAD_FILE}:${MEDIA_TYPE}"
+            if ! retry oras attach --no-tty --registry-config "$HOME/auth.json" --artifact-type "${MEDIA_TYPE}" "${IMAGE_URL}@${IMAGE_DIGEST}" "${UPLOAD_FILE}:${MEDIA_TYPE}"
+            then
+              echo "Failed to attach to ${IMAGE_URL}"
```

Comment:
Is `exit 1` needed after this line?

Reply:
there isn't any set -e so not existing explicitly